clearwater-sip-stress-coreonly.root/usr/share/clearwater/sip-stress/callee.xml

```diff
@@ -19,6 +19,58 @@
 
   <send>
     <![CDATA[
+      SIP/2.0 302 Moved Temporarily
+      [last_Call-ID:]
+      [last_CSeq:]
+      [last_From:]
+      [last_To:];tag=[pid]SIPpTag01[call_number]
+      [last_Via:]
+      Server: SIP/2.0
+      RSeq: 462555543
+      Content-Length: [len]
+      Require: precondition, 100rel
+      [last_P-Charging-Vector:]
+      P-Asserted-Identity: sip:[$dn]@[$home_domain]
+      Contact: <sip:8888000002424@ame22.clearwater.test>
+      Record-Route: <sip:[local_ip]:[local_port];transport=[transport];lr>
+      [last_Record-Route:]
+      Content-Type: application/sdp
+      Allow: INVITE,ACK,OPTIONS,CANCEL,BYE,UPDATE,INFO,REFER,NOTIFY,MESSAGE,PRACK
+      P-Access-Network-Info: 3GPP-E-UTRAN-FDD;utran-cell-id-3gpp=0010100011a2d051
+      User-Agent: BBBBBBB IMS 0.0
+
+      v=0
```

Comment:
Removing the body causes the Contact header to not be sent. Adding body back in after testing.